hazelcast/src/test/java/com/hazelcast/test/TestLoggingUtils.java

```diff
@@ -73,4 +77,39 @@ private static boolean shouldForceTestLoggingFactory() {
         }
         return false;
     }
+
+    public static class CustomTestNameAwareForkJoinPool implements Executor {
+
+        private final Executor defaultExecutor = ForkJoinPool.commonPool();
```

Comment:
Should you still set this conditionally as done at line https://github.com/hazelcast/hazelcast/pull/18181/files#diff-ce99ab5cd3646df5b3e7f7f4c104d12ad68ebed8d5dfdf5f1b5a66430499b539R54

Reply:
No, this is the ForJoinPool implementation that replaces the original one. 
Anything that is send to this Executor is meant to run on `ForkJoinPool.commonPool()`. There is no other executor to run on at this level.